WMass/python/plotter/w-helicity-13TeV/wmass_e/wenu_80X.txt

```diff
@@ -1,8 +1,9 @@
 alwaystrue: 1
-HLT_SingleEL : HLT_SingleEl == 1
+#HLT_SingleEL : HLT_SingleEl == 1
+HLT_SingleEL : HLT_BIT_HLT_Ele27_WPTight_Gsf_v == 1
 onelep : nLepGood == 1 && abs(LepGood1_pdgId)==11
 fiducial : abs(LepGood1_eta)<1.4442 || abs(LepGood1_eta)>1.566
 eleKin : ptElFull(LepGood1_calPt,LepGood1_eta) > 30 && ptElFull(LepGood1_calPt,LepGood1_eta) < 45 && abs(LepGood1_eta)<2.5
-highEtaEE_HLTlowPt: abs(LepGood1_eta) < 2.1 || HLT_BIT_HLT_Ele27_WPTight_Gsf_v == 1 || HLT_BIT_HLT_Ele25_WPTight_Gsf_v == 1
+#highEtaEE_HLTlowPt: abs(LepGood1_eta) < 2.1 || HLT_BIT_HLT_Ele27_WPTight_Gsf_v == 1 || HLT_BIT_HLT_Ele25_WPTight_Gsf_v == 1
```

Comment:
I would remove this commented line, and leave the L2 (because this is the OR of all the single el trigger) (but no strong opinion)